app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -314,11 +314,20 @@ export default {
     // Donot use the keyboard listener mixin here as the events here are supposed to be
     // working even if input/textarea is focussed.
     document.addEventListener('keydown', this.handleKeyEvents);
+    document.addEventListener('paste', this.onPaste);
   },
   destroyed() {
     document.removeEventListener('keydown', this.handleKeyEvents);
+    document.removeEventListener('paste', this.onPaste);
   },
   methods: {
+    onPaste(e) {
+      const data = e.clipboardData.files;
+      if (!data) {
+        return;
+      }
+      this.onFileUpload(data[0]);
+    },
```

Comment:
```suggestion
    onPaste(e) {
      const data = e.clipboardData.files;
      if (!data.length || !data[0]) {
        return;
      }

      const file = data[0];
      const { name, type, size } = file;
      this.onFileUpload({ name, type, size, file });
    },
```

Reply:
We can keep like this to get rid of other changes. All the other changes can be reverted I think.